src/main/cli/cli.c

```diff
@@ -6090,6 +6111,9 @@ const clicmd_t cmdTable[] = {
         "\treset\r\n"
         "\tload <mixer>\r\n"
         "\treverse <servo> <source> r|n", cliServoMix),
+#endif
+#if defined (USE_SERIALRX_SRXL2)
+    CLI_COMMAND_DEF("srxl2_bind", "SRXL2 bind", "[enter|info]", cliSrxl2Bind),
```

Comment:
I think it's a more user friendly approach if we integrate this into `bind_rx_spi` above, and rename the command to `bind_rx` - after all the user will only ever have one RX configured, and should not have to care what RX type it is that they want to bind.
(Thinking ahead, we might want to add this to MSP as well, and give users a 'Bind RX' button in configurator if their RX supports it.)

Reply:
Cli changes for this done in the last commit. Not sure if that was appropriate, or if it should've been a separate pull request. I can undo the changes if so.
Would love to see it in MSP + button in configurator as well.